admin/repositories-jdk6

```diff
@@ -0,0 +1,7 @@
+[repositories]
+local
+local-preloaded-ivy: file:///${sbt.preloaded-${sbt.global.base-${user.home}/.sbt}/preloaded/}, [organization]/[module]/[revision]/[type]s/[artifact](-[classifier]).[ext]
+local-preloaded: file:///${sbt.preloaded-${sbt.global.base-${user.home}/.sbt}/preloaded/}
+maven-central-over-http: http://repo1.maven.org/maven2/
```

Comment:
so there's also an issue with downloading from maven central, which is worked around here? this doesn't fix publishing to sonatype, right? could you explain in the commit message?

Reply:
Correct, we should also change the publishTo to http.

https://github.com/scala/sbt-scala-module/blob/5c7c76c003cf681ca8d3199bec4cc8b8477942eb/src/main/scala/ScalaModulePlugin.scala#L117-L119

It's using:

```
> show compat/publishTo
[info] Some(sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots)
```

However, this would be a security vulnerability. I will close this PR. The other solution to use jdk8 for sbt but use annother jdk forking makes sense.